main.tf

```diff
@@ -202,6 +244,7 @@ module "additional_tfe_workspaces" {
   terraform_version              = each.value.terraform_version != null ? each.value.terraform_version : var.tfe_workspace.terraform_version
   trigger_prefixes               = each.value.connect_vcs_repo != false ? coalesce(each.value.trigger_prefixes, var.tfe_workspace.trigger_prefixes) : null
   username                       = coalesce(each.value.username, "TFEPipeline-${each.key}")
+  variable_set_ids               = { "account" : tfe_variable_set.account.id }
```

Comment:
you need to merge this with each.value.variable_set_ids

Reply:
Ah, yes!

Reply:
Fixed.